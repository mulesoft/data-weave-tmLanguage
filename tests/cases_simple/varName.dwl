%dw 2.0
var aaaa = 1
ns aa bb
var BBBB = 2
var asd = null
type XXXX = YYYY
fun XXXX<T>(): Number<X> = YYYY as String
---