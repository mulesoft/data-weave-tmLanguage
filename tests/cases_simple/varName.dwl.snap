>%dw 2.0
#^^^ source.data-weave meta.directive.version.dw keyword.directive.version.dw
#   ^ source.data-weave meta.directive.version.dw
#    ^^^ source.data-weave meta.directive.version.dw constant.numeric.version.dw
>var aaaa = 1
#^^^ source.data-weave storage.type.dw
#   ^ source.data-weave
#    ^^^^ source.data-weave entity.name.variable.dw
#        ^ source.data-weave
#         ^ source.data-weave keyword.operator.assignment.dw
#          ^ source.data-weave
#           ^ source.data-weave constant.numeric.dw
>ns aa bb
#^^ source.data-weave storage.type.dw
#  ^ source.data-weave
#   ^^ source.data-weave entity.name.variable.dw
#     ^ source.data-weave
#      ^^ source.data-weave markup.underline.link
>var BBBB = 2
#^^^ source.data-weave storage.type.dw
#   ^ source.data-weave
#    ^^^^ source.data-weave entity.name.variable.dw
#        ^ source.data-weave
#         ^ source.data-weave keyword.operator.assignment.dw
#          ^ source.data-weave
#           ^ source.data-weave constant.numeric.dw
>var asd = null
#^^^ source.data-weave storage.type.dw
#   ^ source.data-weave
#    ^^^ source.data-weave entity.name.variable.dw
#       ^ source.data-weave
#        ^ source.data-weave keyword.operator.assignment.dw
#         ^ source.data-weave
#          ^^^^ source.data-weave constant.language.dw
>type XXXX = YYYY
#^^^^ source.data-weave storage.type.dw
#    ^ source.data-weave
#     ^^^^ source.data-weave entity.name.variable.dw
#         ^ source.data-weave
#          ^ source.data-weave keyword.operator.assignment.dw
#           ^ source.data-weave
#            ^^^^ source.data-weave entity.name.variable.dw
>fun XXXX<T>(): Number<X> = YYYY as String
#^^^ source.data-weave storage.type.dw
#   ^ source.data-weave
#    ^^^^ source.data-weave entity.name.variable.dw
#        ^ source.data-weave keyword.operator.comparison.dw
#         ^ source.data-weave entity.name.variable.dw
#          ^ source.data-weave keyword.operator.comparison.dw
#           ^^ source.data-weave
#             ^ source.data-weave keyword.operator.declaration.dw
#              ^ source.data-weave
#               ^^^^^^ source.data-weave entity.name.variable.dw
#                     ^ source.data-weave keyword.operator.comparison.dw
#                      ^ source.data-weave entity.name.variable.dw
#                       ^ source.data-weave keyword.operator.comparison.dw
#                        ^ source.data-weave
#                         ^ source.data-weave keyword.operator.assignment.dw
#                          ^ source.data-weave
#                           ^^^^ source.data-weave entity.name.variable.dw
#                               ^ source.data-weave
#                                ^^ source.data-weave keyword.operator.dw
#                                  ^ source.data-weave
#                                   ^^^^^^ source.data-weave entity.name.variable.dw
>---
#^^^ source.data-weave keyword.operator.body-marker.dw